job/main.py

```diff
@@ -0,0 +1,63 @@
+import os
+import analytics
+import pandas as pd
+from analytics import Client
+from google.cloud import automl_v1beta1
+import stacklogging
+
+logger = stacklogging.getLogger(__name__)
+
+MODEL_NAME = "trial_prediction_model"
+MODEL_VERSION = "v5"
+
+segment_client = Client(os.getenv("SEGMENT_WRITE_KEY"), max_queue_size=100000)
+automl_client = automl_v1beta1.TablesClient(project="buffer-data", region="us-central1")
+
+# Run the batch predict
+logger.info("Running AutoML Batch predict")
+
+operation = automl_client.batch_predict(
+    bigquery_input_uri="bq://buffer-data.dbt_buffer.predict_publish_trial_conversion_holdout",
```

Comment:
Hey @davidgasquez! I was curious why we're using the `predict_publish_trial_conversion_holdout` table here instead of `predict_publish_trial_conversion_prediction`? In [this PR](https://github.com/bufferapp/buffer-dbt/pull/45), I was intending to use the `predict_publish_trial_conversion_prediction` model as the source for our batch predictions (not sure if I'm missing something though).

Reply:
Ups! We're using it becuase I copied the wrong URI. :see_no_evil: 

We should be using `predict_publish_trial_conversion_prediction`.

Reply:
Fixed in https://github.com/bufferapp/convertice/commit/dcdd6ece20179b4ad3bc0fb44199939baace4d2a!

Reply:
Awesome, thanks David!

Reply:
Awesome, thanks David!